worlds/ff4fe/__init__.py

```diff
@@ -191,6 +214,11 @@ def set_rules(self) -> None:
                         continue
                 add_item_rule(self.get_location(location.name),
                               lambda item: item.name not in items.characters)
+                if self.options.ItemPlacement.current_key == "major_minor_split" and location.major_slot == False:
```

Comment:
this can be rewritten to 
```python
if self.options.ItemPlacement.current_key == "major_minor_split" and not location.major_slot:
```

Also, a check to see if this location is in priority locations should be added here, since forbidding a priority location from having any progression items will cause a 100% generation failure rate.